src/essence/Basics/Layers_/Layers_.js

```diff
@@ -1937,6 +1945,26 @@ var L_ = {
         // Clear the description
         Description.clearDescription()
     },
+    //Takes in a config layer object
+    globeLithoLayerHelper: async function(s) {
+        if (L_.Globe_) {
+            // Hold the previous layer state
+            const prevState = L_.toggledArray[s.name]
+
+            // Only toggle the layer to reset if the layer is toggled on,
+            // because if the layer is toggled off, it is not on the globe
+            if (prevState) {
+                // Temporarily set layer to on
+                L_.toggledArray[s.name] = false
+
+                // Toggle the layer so its drawn in the globe
+                await L_.toggleLayer(s)
+
+                // Reset the previous layer state
+                L_.toggledArray[s.name] === prevState
```

Comment:
Can we try to get rid of this line? `L_.toggleLayer()` should already be doing this on its own terms.  

Also, if `L_.toggleLayer()` doesn't need the await, I'd get rid of it.

Reply:
Fixed in 5f38e6800938884468eefc5b3bb1c2f1452dee52!